src/pages/mesh/basic/create-mesh.md

```diff
@@ -77,11 +81,25 @@ When creating or updating a mesh, the file to upload must have the `.json` filen
 
 1. When you are prompted to confirm that you want to create a mesh, select **Yes**. If you want to automatically confirm the creation, add the `-c` or `--autoConfirmAction` flag to your create command.
 
-  The `aio api-mesh:create` response assigns you a `meshId`. Use the [`aio api-mesh:status`](../advanced/index.md#aio-api-meshstatus) command to see the status of your mesh creation. You can run the [`aio api-mesh:describe`](../advanced/index.md#aio-api-meshdescribe) command to get your `apiKey` and a GraphQL endpoint that you can use to query your mesh.
+### Access your mesh URLs
+
+The `aio api-mesh:create` response assigns you a `meshId`. Use the [`aio api-mesh:status`](../advanced/index.md#aio-api-meshstatus) command to see the status of your mesh creation. You can run the [`aio api-mesh:describe`](../advanced/index.md#aio-api-meshdescribe) command to get your `apiKey` and a GraphQL endpoint that you can use to query your mesh.
+
+Until Adobe deprecates legacy mesh URLs, the response to the `aio api-mesh:describe` command will list both the legacy and edge mesh URLs. The legacy mesh URL is a standard mesh URL, which Adobe will deprecate later this year. The edge mesh URL offers improved response times. API Mesh stores edge meshes closer to your data sources to reduce latency and generally improve the API Mesh experience by making meshes more performant. Edge meshes provide the following advantages:
```

Comment:
Can we clean this up (and reduce our legal exposure) with some slight changes. We already list the advantages in the bullets so I don't see a strong reason to say the same things in this paragraph. I also want to limit promises of improved perf over MGQL. 

"The edge mesh URL offers several benefits as edge meshes are stored closer to your data sources. Edge meshes can provide the following advantages: